examples/catalog/src/Reset.scss

```diff
@@ -0,0 +1,77 @@
+html, body, div, span, applet, object, iframe,
```

Comment:
How did you come up with this list? I think you only need `html, body`

Reply:
I follow this guide from Eric Meyer's reset.css
Of course I don't use all of tag.
But I didn't touch it because it looks like a library file.